src/Microsoft.ML.TimeSeries/SequentialTransformerBase.cs

```diff
@@ -66,7 +69,19 @@ protected long IncrementRowCounter()
                 return RowCounter;
             }
 
-            private bool _isIniatilized;
+            protected long PreviousPosition;
+
+            protected StateBase(ModelLoadContext ctx)
```

Comment:
>protected [](start = 12, length = 9)

`protected` is better than `publi`c. Private protected is better than protected.

Remember, we're trying to reduce the surface area of our API, and a protected member is still parse of the API surface area...